tests/test_docs/helper.py

```diff
@@ -188,26 +186,14 @@ def extract_make_commands(makefile_paths: List[str]) -> List[str]:
                 match = re.match(MAKEFILE_COMMAND, line)
                 if match:
                     commands += match.groupdict()["command"].split(" ")
+                    continue
+                match = re.match(MAKEFILE_COMMAND_PHONY, line)
+                if match:
+                    commands += match.groupdict()["command"].split(" ")
     return commands
 
 
-def extract_autonomy_commands() -> List[str]:
-    """Extract autonomy commands from the autonomy cli"""
-    cmd_list = []
-    for cmd_name, cmd in cli.commands.items():
```

Comment:
We now have the new validity test. It doesn't make sense to have this primitive version here anymore.

Reply:
I am not following. Can you reference the relevant PRs?

Reply:
#1322 , #1343 , #1354 Implemented the command validator test that searches for commands in docs and Makefile, making the test below (check_bash_commands_exist, which uses this function) partially obsolete, since this way of checking autonomy commands is pretty basic.